File: src/main/java/cc/mirukuneko/realtrainmodrenewed/client/ClientNetworkHelper.java
package cc.mirukuneko.realtrainmodrenewed.client;

import net.minecraft.client.Minecraft;
import net.minecraft.network.protocol.common.custom.CustomPacketPayload;

public final class ClientNetworkHelper {
    private ClientNetworkHelper() {
    }

    public static void sendToServer(CustomPacketPayload payload, CustomPacketPayload... ignored) {
        var connection = Minecraft.getInstance().getConnection();
        if (connection != null) {
            connection.send(payload);
        }
    }
}

File: src/main/kotlin/cc/mirukuneko/realtrainmodrenewed/client/ClientNetworkHelper.kt
package cc.mirukuneko.realtrainmodrenewed.client

import net.minecraft.client.Minecraft
import net.minecraft.network.protocol.common.custom.CustomPacketPayload

object ClientNetworkHelper {
    @JvmStatic
    fun sendToServer(payload: CustomPacketPayload, vararg ignored: CustomPacketPayload) {
        Minecraft.getInstance().connection?.send(payload)
    }
}
